src/content/components/button/style.md

```diff
@@ -103,3 +102,12 @@ A Button cannot have any element or text within 16 pixels / 1 rem of its borders
   </div>
 </div>
 _Spacing measurements for various Button types | px / rem_
+
+### Recommended
+
+The following specs are not built into any of the Button components but are recommended by design as the proper amount of space to distance Buttons should be from one another.
```

Comment:
I think there's something off at the end of this sentence.

Reply:
Fixed!